lib/view/CsrBB.class.php

```diff
@@ -300,7 +300,7 @@ function bb_neuzen($arguments = array()) {
 			$content = $arguments;
 		}
 		if (LidInstellingen::get('layout', 'neuzen') != 'nee') {
-			$neus = '<img src="/plaetjes/famfamfam/bullet_red.png" alt="o" class="neus2013">';
```

Comment:
> en krijg je gewone tekst terug.

Dat is helemaal fijn! Kun je dat ook fixen voor de visitekaartjes in DataTable rows?